modulation.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np 
+
+class modulation:
+    def __init__(self, qm, bit_number):
```

Comment:
Treba omogućiti da konstrutkor može da radi i ako je dat samo jedan od parametara - odnosno da se onaj drugi preračuna. Ukoliko se daju oba parametra to treba hendlovati na odgovarajući način.